### ISSUE TITLE
[FEATURE]: TorchDiceLoss logits

### ISSUE THREAD

**Issue by dphogan** [CosmiQ/solaris#281]
Some loss functions, such as TorchFocalLoss, take a flag argument for whether or not to apply a sigmoid function to the input.  It would be good to add this feature to TorchDiceLoss in _torch_losses.py.  Although one can sometimes render this unnecessary by taking the sigmoid in the model's forward function instead of in the loss function, having this option in the loss function is useful for when using a weighted sum of loss functions.

**Comment by dphogan** [comment]
Resolved with pull #283
